### DISCUSSION TITLE
fun to

### DISCUSSION THREAD

**Discussion by Kidus-fu** [community/community#133020] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hi i'm kidus i'm web back-end devlopment student

**Comment by SamuelOUS** [comment]
Hi Kidus :)

**Reply by Kidus-fu** [reply]
hi samuelOUS nice to meet you bro :)

**Comment by ebndev** [comment]
Welcome to GitHub and the GitHub Community @Kidus-fu 🎆! We're glad you're here and can't wait to see where your GitHub journey takes you.

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and connect with other users there! 👋

**Reply by Kidus-fu** [reply]
ok bro appreciate
